package.json

```diff
@@ -56,19 +54,27 @@
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.2.0",
     "eslint-plugin-import": "^2.18.2",
+    "husky": "^3.1.0",
     "jest": "^24.9.0",
     "nhsuk-frontend": "^3.0.2",
     "nock": "^10.0.6",
     "node-sass": "^4.13.0",
     "sass-lint": "^1.13.1",
     "supertest": "^4.0.2",
+    "testcafe": "^1.6.1",
+    "testcafe-reporter-nunit": "^0.1.2",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.10"
   },
   "jest": {
     "testPathIgnorePatterns": [
       "/node_modules/",
-      "<rootDir>/integration-tests"
+      "((\\.|/)(ui.test))\\.js$"
```

Comment:
WOW how did figure that one out

Reply:
google ...then working backwards from the regex that jest uses to find the files in the first place :)